### PR TITLE
fix: Inpatient Record - Allow user to create inpatient record

### PR DESCRIPTION
**Pull request by akashkrishna619**
- Allow user to create inpatient record directly ([Issue link](https://discuss.erpnext.com/t/no-save-button-for-inpatient-record-doc/88494))
- Cancel Admission option for admission scheduled inpatient records
- Remove inpatient_record link fetching from vital signs in get_patient_detail method

### CODE REVIEW

File: healthcare/healthcare/doctype/inpatient_record/inpatient_record.js [hunk 1]
```diff
@@ -11,28 +11,52 @@ frappe.ui.form.on('Inpatient Record', {
...
-		}
+		frm.set_query('primary_practitioner', function() {
+			return {
+				filters: {
+					'department': frm.doc.medical_department
+				}
+			};
+		});
+		if (!frm.doc.__islocal) {
+			if (!frm.doc.admission_encounter) {
```
Comment (ChillarAnand) [inline healthcare/healthcare/doctype/inpatient_record/inpatient_record.js]: Club both if statements?

**Comment by ChillarAnand** [comment]
Can you please update relevant docs at https://docs.erpnext.com/docs/v13/user/manual/en/healthcare/inpatient_record?

**Comment by mujeerhashmi** [comment]
@akashkrishna619 @ChillarAnand It is more intuitive to add the **schedule Admission** & **Schedule Discharge** button to Patient doctype based on patient status. Right now we have these buttons in Patient Encounter. This will enable the direct admission from patient doctype.

**Comment by ChillarAnand** [comment]
Generally, staff at the admission counter will have Inpatient record page open so that they can process admissions.

From Patient doctype, there is a connection to `Inpatient Record`. Users can directly go there and add relevant details? @mujeerhashmi 

Screenshot 2022-04-28 at 14 50 59

If a patient has relevant inpatient record, may be we can show a label on the patient doctype.

**Comment by mujeerhashmi** [comment]
> From Patient doctype, there is a connection to `Inpatient Record`. Users can directly go there and add relevant details? @mujeerhashmi

Ok. This will also do. But like I said It is more intuitive to have that button that allows adding initial details for IP record creation. Which is generally done at the counter where patient registration happens. We have done this for a multi-specialty hospital in a tier-2 city.

File: healthcare/healthcare/doctype/inpatient_record/inpatient_record.js (hunk 1 above)
Comment (akashkrishna619) [reply healthcare/healthcare/doctype/inpatient_record/inpatient_record.js]: yes thanks, I've made some minor changes
- If statements clubbed
- Moved 'Admit' button show/hide part to json display_depends_on

**Review by ChillarAnand — APPROVED** [review]

**Pull request merged**